tendrl/node_agent/node_sync/check_all_managed_nodes_status.py

```diff
@@ -1,20 +1,54 @@
 import etcd
 
+from tendrl.commons.objects.job import Job
+import uuid
+
+
+def update_brick_status(fqdn, integration_id):
```

Comment:
I dont think this belongs here, please add this to https://github.com/Tendrl/node-agent/tree/master/tendrl/integrations/gluster

check
https://github.com/Tendrl/node-agent/tree/master/tendrl/integrations/ceph  (for more details)

Reply:
How will gluster_integration figure out about the node going down ?
Here node_agent "monitor" node already keeps track of any node going down and fires a job for "gluster_integraion" to update the status.

Reply:
I meant to move the from here to https://github.com/Tendrl/node-agent/tree/master/tendrl/integrations/gluster/sds_sync/\__init\__.py 

It will run in the node-agent process itself, you will have to start the above sync like here https://github.com/Tendrl/node-agent/blob/master/tendrl/node_agent/manager/__init__.py#L97

Reply:
This is the only place where we get to know the point in time when node went down. If we move it to the place you suggested(in fact I tried that and during verification realized till the node comes up we will continuously fire jobs on every sync) we will end up firing jobs to mark bricks as down on every sync
Pls suggest

Reply:
@r0h4n Please provide your inputs

Reply:
@nnDarshan Firstly, Copy this entire function (including node status check) to https://github.com/Tendrl/node-agent/tree/master/tendrl/integrations/gluster/sds_sync/%5C__init%5C__.py  as I had mentioned in earlier comment

And in update_brick_status(), if brick status is down, dont send the job

Reply:
@r0h4n If I am checking the brick status before firing the job, I can mark the status as down from here itself. why bother about the job.
What do you feel ?

Reply:
@nnDarshan We need the job, the brick object is owned (any changes) by Gluster namespace. Please call the flow